api/models/mmis.js

```diff
@@ -16,17 +16,17 @@ const mmisSchema = new mongoose.Schema(
         asNeededUpdate: Boolean
       },
       medicaidBusinessAreas: {
-        waiverSupportSystems: Boolean,
-        assetVerificationSystem: Boolean,
+        waiverSupport: Boolean,
```

Comment:
you might want to make the change the other way around, because you're going to need a migration if you change the model

Reply:
sounds good. I should have thought of that 🤦